### PR TITLE
Analytics, embedded map, event info

### PR DESCRIPTION
**Pull request by flagoworld**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,6 +19,14 @@ <h5 class="header col s12 light">
           Who are you?
           </a>
       </div>
+      <div class="row center">
+          <br /><br /><br />
```
Comment (awaterma) [inline index.html:23]: Could you replace the br's and span tag with another div here?

File: survey.html [hunk 2]
```diff
@@ -159,4 +159,16 @@
     });
   </script>
 </body>
+
+<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```
Comment (awaterma) [inline survey.html:164]: Do we need to also publish a special http page for google? Or will this work standalone? :)

Comment (flagoworld) [inline survey.html:164]: What do you mean "special http page"?

File: index.html (hunk 1 above)
Comment (flagoworld) [inline index.html:23]: No idea. I just saw that <br />s were used as spacers up above and I didn't really want to dive into the CSS.

**Comment by awaterma** [comment]
Well, that was lame of me! I'll pull the br's when I get back home. :) I'm traveling this week, so no access to my machine. I can merge this, though, pull from IslandJS and update both locations when I'm back.

In terms of Google, I had thought we had to register the domain with them before we can get stats, but It looks like I'm thinking of webmaster tools: https://support.google.com/webmasters/answer/35179?hl=en

I'll merge.

**Pull request merged**